### DISCUSSION TITLE
Tell me about your favorite songs!

### DISCUSSION THREAD

**Discussion by Akari000** [NAIST-SE/AttractingContributors#476] [General]
Do you have songs you listen to often or are nostalgic?
Please tell me the name of your favorite song!
よく聞く曲や思い出の曲はありますか？
お気に入りの曲を教えてください！

**Example**
1. 金木犀の夜
2. クロノスタシス (Just write the name of the song!)

---
**The repository is here.**
**https://github.com/Akari000/AttractingContributors-favorite-song**

**Comment by XC1996114514** [comment]
I have filled the readme.

Please help me if you'd like to 🥺🥺🥺
Thanks a lot for your help


https://github.com/XianChen1996/AttractingContributors_type_in_your_favourite_footballteam

Add to my README file.

I look forward to your contributions

**Comment by ploychanok** [comment]
I created a pull request on your repo.
Would you mind help mine too? https://github.com/ploychanok/AttractingContributors-WithinThreeWords

**Comment by tmymzkr** [comment]
I done pull request 
please help me
https://github.com/m-tomoya/AttractingContributors-movie
